src/components/TutorialCarouselDialog/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, withStyles } from 'material-ui';
+import Dialog, { DialogContent } from 'material-ui/Dialog';
+import cx from 'classnames';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import _ from 'lodash';
+
+import styles from './styles';
+import graphqlActions from '../../redux/Graphql/actions';
+import Tutorial0 from './components/tutorial0';
+import Tutorial1 from './components/tutorial1';
+import Tutorial2 from './components/tutorial2';
+import Tutorial3 from './components/tutorial3';
+import Tutorial4 from './components/tutorial4';
+import Tutorial5 from './components/tutorial5';
+import Tutorial6 from './components/tutorial6';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+export default class TutorialCarouselDialog extends Component {
```

Comment:
since we're not using `state` here, can we please make this a stateless component

Reply:
actually we are using state here.